lighthouse-extension/app/src/popup.js

```diff
@@ -37,6 +37,32 @@ document.addEventListener('DOMContentLoaded', _ => {
   const optionsList = document.body.querySelector('.options__list');
   const okButton = document.getElementById('ok');
 
+  function getLightHouseVersion() {
+    return chrome.runtime.getManifest().version;
+  }
+
+  function getChromeVersion() {
+    return /Chrome\/([0-9.]+)/.exec(navigator.userAgent)[1];
+  }
+
+  function buildReportErrorLink(err) {
+    const reportErrorEl = document.createElement('a');
+    reportErrorEl.className = 'button button--report-error';
+
+    let qsBody = '**Lighthouse Version**: ' + getLightHouseVersion() + '\n';
+    qsBody += '**Chrome Version**: ' + getChromeVersion() + '\n';
+    qsBody += '**Stack Trace**:\n' + err.stack;
```

Comment:
Good call and done. After placing it in a code block, I noticed the original error wasn't in the stack trace, so I also added that.